README.md

```diff
@@ -51,22 +51,30 @@ There are different ways of creating providers inside Pulumi but for this projec
 
 ## How to Use
 
-1. Create you Pulumi project file by copying the example (`cp Pulumi.example.yaml Pulumi.yaml`) and setting name, runtime (keep as it is) and description.
+Before starting make sure you login with your Pulumi CLI. 
 
-3. For CI/CD environments, you need to add same environment variables to your secrets in your system. In the case of `GitHub Actions`, you can use `Secrets` as described [here](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets). The mapping is as following:
+1. Create you Pulumi project file by copying the example (`cp Pulumi.example.yaml Pulumi.yaml`) and setting `name`, `runtime` (keep as it is) and `description`.
 
-- PULUMI_ACCESS_TOKEN
-- TWILIO_<BRANCH_NAME>_ACCOUNT_SID
-- TWILIO_<BRANCH_NAME>_AUTH_TOKEN
+2. For development environment, you should copy the .dev.env.example (`cp .dev.env.example .dev.env`) and fill out the variables: 
 
-2. For development environment, you should copy the .dev.env.example (`cp .dev.env.example .dev.env`) and fill out the variables. Also, do not forget to add `.dev.env` files to all your serverless directories. After that, you can run the package scripts without the `ci:` in the beginning of their names. For example: 
+* `FLEX_WORKSPACE_SID`: This is the SID of the "Flex Task Assignment" TaskRouter Workspace in your Flex project
+* `BRANCH_NAME`: This is the name you want to give to the Pulumi stack (e.g. `dev`)
+
+3. Create two files called `.<BRANCH_NAME>.env` in  `serverless/autopilot` and `serverless/main`, where `BRACH_NAME` is the value of the variable set above. These files holds configuration for autoppilot and serverless. For this example you can leave them blank. 
```

Comment:
There are some typos here such as `BRACH_NAME`and `autoppilot`

Reply:
Also, I made a change in the `twilio-pulumi-provider` where in v0.0.8 we don't need to keep the .`<BRANCH_NAME>.env` files in serverless directories (`serverless/autopilot` and `serverless/main`) if no actual environment variable is inside. What do you think of rephasing this part and updating the package.json to use the v0.0.8 of the `twilio-pulumi-provider`?

Reply:
So, should we add a `(optional)` in front of step 3? I haven't tried v0.0.8 but what would happen if I add an `.env` in the subfolder? Would that be used?